samples/cellular/modem_shell/src/at/at_cmd_mode_sett.c

```diff
@@ -64,6 +72,29 @@ bool at_cmd_mode_sett_is_autostart_enabled(void)
 	return sett_autostart_enabled;
 }
 
+int at_cmd_mode_sett_echo_enabled(bool enabled)
```

Comment:
I'm wondering if we should make this setting persistent or not. You've already implemented it, but of previous settings only `autostart` is persistent. Obviously that has to be persistent to work, but the LF mode setting does not persist. Also, the setting configured with the `ATE` command is stored immediately. There are some AT commands where the configuration is stored into NV memory, but even those require modem to be shutdown using `AT+CFUN=0` for the setting to be stored.

Reply:
In my use case it helps having this setting persistent because of the third-party PC application behaviour. I have no strong opinion when to store, I can always move this to a `CFUN=0` hook.

Reply:
Not many lines of code.